### ISSUE TITLE
Failing test: tests/plugins/autocomplete/manual/a11ydivarea

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#4704]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Open `tests/plugins/autocomplete/manual/a11ydivarea` test and follow the test scenario

### Expected result

The test passed. The same way as described in the test scenario.

### Actual result

JAWS didn't announce "has popup" or "has autocomplete" in step `2`
JAWS announce "listbox at john" when listbox was opened. Since "@john" is the first option
JAWS announce while switching options, "at thomas" or "at anna" without e.g. "2 of 5"
JAWS didn't announce that the listbox is collapsed at step `5`. It announces the same sentence when you focus the editor in step `2`

## Other details

* Browser: IE11
* OS: Windows 10
* CKEditor version: 4.16.1
* Installed CKEditor plugins: -

**Comment by f1ames** [comment]
Could be a follow-up of #4617. I would say this is a feature request since [JAWS + IE11 is not officially supported](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_browsers.html#accessibility-support).
